frontend/src/components/footer/Footer.js

```diff
@@ -15,5 +15,11 @@ export default function Footer({ children }){
 	const isLarge = useMediaQuery({ query: "(min-width: 600px)" });
 	const FooterComponent = isLarge ? LargeFooter : SmallFooter;
 
-	return <FooterComponent>{children}</FooterComponent>;
+	let mainStyle = isLarge ? { minHeight: "calc(100vh - 220px)", paddingBottom: "220px" } : { minHeight: "calc(100vh - 220px)", paddingBottom: "220px" };
+
+	return (<>
```

Comment:
What is the advantage of these `<> ... </>` ?

Reply:
Read this: https://reactjs.org/docs/fragments.html. The empty tag is just the short syntax of it.

Reply:
Cool! I thought about this fragment problem as well but didn't know, this is the solution :-) .